scripts/gke/deployment_manager_configs/cluster-kubeflow.yaml

```diff
@@ -53,11 +53,9 @@ resources:
       # Whether to enable Pod Security Policy Admission Controller
       # Use v1beta1 api
       podSecurityPolicy: false
-      masterAuthorizedNetworksConfig:
-          cidrBlocks:
```

Comment:
just curious: what's this?

Reply:
By default any IP is allowed to connect to the master. `masterAuthorizedNetworksConfig` is a fancy name for firewall rules for master.